dnf-behave-tests/dnf/plugins-core/enable-plugins-dashes.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Tests for enabling plugins containing dashes in names
+
+Background: Enable plugins
+  Given I do not disable plugins
+    And I use repository "dnf-ci-fedora"
+
+@bz1980712
+Scenario: Existing plugin containing dashes in the name could be disabled
+   When I execute dnf with args "repoquery empty --disableplugin=config-manager --verbose"
+   Then the exit code is 0
+    And stdout does not contain "config-manager"
```

Comment:
I would prefer more accurate check here. The exact match would require defined set of available plugins, but at least something like `And stdout does not contain "Loaded plugins: .*config-manager"` would be better. Or maybe `stdout matches line by line` step might be useful (at least for positive cases).

Reply:
Definitely, this is better and I didn't check that `contain` step is using pattern matching.